__init__.py

```diff
@@ -360,16 +399,19 @@ def load_credentials(self):
         except HTTPError:
             LOG.info('Couldn\'t fetch credentials')
             self.spotify = None
+            return
 
-        if self.spotify:
-            # Spotfy connection worked, prepare for usage
-            # TODO: Repeat occasionally on failures?
-            # If not able to authorize, the method will be repeated after 60
-            # seconds
-            self.create_intents()
-            # Should be safe to set device_name here since home has already
-            # been connected
-            self.device_name = DeviceApi().get().get('name')
+        if not self.spotify:
+            self.speak_dialog('NotConnected')
```

Comment:
I don't think this makes sense since this is in a scheduled routine and will be triggered periodically in the background